src/Invitation/BrowseTheme.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { ProgressBar, Container, Row, Col, Image } from 'react-bootstrap';
+import BlueButton from '../Common/styled/buttonstyles/BlueButton';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+export default function BrowseTheme() {
+  return (
+    <Container className="mt-5">
+      <Row>
+        <Col xs lg="9">
+          <ProgressBar now={50} />
+          <h2>Wedding Website</h2>
```

Comment:
When you get the chance to do more styling, take a look at the common/styled/textstyles folder, all the headers and text you need should be in there, let me know if you need to add one!